Readme.md

```diff
@@ -9,7 +9,7 @@ Easy JavaScript vs WebAssembly performance comparison test-set.
 ## Develop locally
 
 ```sh
-$ git clone https://github.com/takahirox/WebAssembly-benchmark.git
+$ git clone https://github.com/mavrukin/WebAssembly-benchmark.git
```

Comment:
I guess this line shouldn't be changed. Or any special reasons why you want users to clone from your fork?